feat: Add the auth.yaml file and add server port information

leesuin0710 opened the pull request:
## ✅ 구현된 기능
- configs 폴더 아래 auth.yaml 파일 추가
- auth.yaml에 서버 포트 29080 설정 추가

---

## 🚨 리뷰 포인트
- auth 서버 정상 구동 확인

leesuin0710 committed:
feat: Add the auth.yaml file and add server port information

chocolaggibbiddori commented on configs/auth.yaml:
파일 이름을 application.yaml로 해주시고, configs 하위에 dollar-auth 디렉토리 만들어서 넣어주세요.

leesuin0710 committed:
fix: Modify the auth configuration file structure and contents
- create auth folder
- delete auth.yaml
- creating the application.yaml file and write the auth server settings
